src/services/db/GitHubService.ts

```diff
@@ -286,30 +357,46 @@ export default class GitHubService {
       return { newSha: resp.data.content.sha }
     } catch (err) {
       if (err instanceof NotFoundError) throw err
-      const { status } = err.response
-      if (status === 404) throw new NotFoundError("File does not exist")
-      if (status === 409)
-        throw new ConflictError(
-          "File has been changed recently, please try again"
-        )
-      throw err
+      if (axios.isAxiosError(err)) {
```

Comment:
We have a `isAxiosError` util method, are we able to use that one?

Reply:
b7fa30138bb1b5bacb5ff10b367e6c6d0d81473e